ganlib/classifier.py

```diff
@@ -91,8 +92,116 @@ def from_checkpoint(cls, ckpt, *args, **kwargs):
     def to_checkpoint(self, ckpt):
         torch.save(self.state_dict(), ckpt)
 
+<<<<<<< HEAD
+choices = {
+    'mnist': MNIST,
+    'wide_resnet':WideResNet,
+    'cifar10': CIFAR10
+}
+
+
+
+""" Copyright 2020 Vithursan Thangarasa Licensed under the Educational Community License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
```

Comment:
We don't want to copy-paste.  We want to reproduce, so that this license does not apply.  We'll give reference to his work, but we'll use our own code.